packages/postcss-convert-values/src/index.js

```diff
@@ -106,7 +106,12 @@ function transform(opts, decl) {
           });
           return false;
         }
-        if (lowerCasedValue === 'url') {
+        if (
+          lowerCasedValue === 'url' ||
+          lowerCasedValue === 'min' ||
+          lowerCasedValue === 'max' ||
+          lowerCasedValue === 'clamp'
```

Comment:
We need to look on CSS spec for all possible values

Reply:
`min()`, `max()` and `clamp()` were introduced in [CSS Values Module Level 4](https://www.w3.org/TR/css-values-4/#calc-notation), although I don't know if there are other CSS functions in other modules that don't accept unitless `0`.

Reply:
Hm, what about calc?

Reply:
I've had a look and units on 0 values are preserved already in `calc()` — maybe these should be moved to the same place?

Reply:
It is strange, I think logic for calc should be here too

Reply:
I think that would introduce a lot of regressions in the current transformations in `calc`.

How about something like this to bring the functions together:

```diff
         if (
           lowerCasedValue === 'calc' ||
+          lowerCasedValue === 'min' ||
+          lowerCasedValue === 'max' ||
+          lowerCasedValue === 'clamp' ||
           lowerCasedValue === 'hsl' ||
           lowerCasedValue === 'hsla'
         ) {
           walk(node.nodes, (n) => {
             if (n.type === 'word') {
               parseWord(n, opts, true);
             }
           });
           return false;
         }
-        if (
-          lowerCasedValue === 'url' ||
-          lowerCasedValue === 'min' ||
-          lowerCasedValue === 'max' ||
-          lowerCasedValue === 'clamp'
-        ) {
+        if (lowerCasedValue === 'url') {
           return false;
         }
       }
```

Reply:
Yes, let's do it

Reply:
New pull request?

Reply:
Yep